cli/mobycli/scout_suggest.go

```diff
@@ -53,10 +56,14 @@ func displayScoutQuickViewSuggestMsg(image string) {
 	}
 	out := os.Stderr
 	b := color.New(color.Bold)
-	_, _ = fmt.Fprintln(out)
-	_, _ = b.Fprintln(out, "What's Next?")
-	_, _ = fmt.Fprintf(out, "  View summary of image vulnerabilities and recommendations → %s", color.CyanString("docker scout quickview%s", image))
-	_, _ = fmt.Fprintln(out)
+
+	_, _ = b.Fprintln(out, "\nWhat's Next?")
+	if !hubLoggedIn() {
+		_, _ = fmt.Fprintf(out, "  1. Sign in to your Docker account → %s\n", color.CyanString("docker login"))
+		_, _ = fmt.Fprintf(out, "  2. View a summary of image vulnerabilities and recommendations → %s\n", color.CyanString("docker scout quickview%s", image))
+	} else {
+		_, _ = fmt.Fprintf(out, "  View summary of image vulnerabilities and recommendations → %s\n", color.CyanString("docker scout quickview%s", image))
```

Comment:
If we're nit-picking; can save some trees here 🌴 🌳 🌲 

```suggestion
	if !hubLoggedIn() {
		_, _ = fmt.Fprintln(out, "  1. Sign in to your Docker account → "+color.CyanString("docker login"))
		_, _ = fmt.Fprintln(out, "  2. View a summary of image vulnerabilities and recommendations → "+color.CyanString("docker scout quickview"+image))
	} else {
		_, _ = fmt.Fprintln(out, "  View a summary of image vulnerabilities and recommendations → "+color.CyanString("docker scout quickview"+image))
```